<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tienda de Videojuegos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Imagenes/mando.png</normaloff>Imagenes/mando.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="cestaPedido">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>631</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Comprar">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>620</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
background-color: rgb(0, 51, 102);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Comprar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Imagenes/comprar.png</normaloff>Imagenes/comprar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancelar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
background-color: rgb(0, 51, 102);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Imagenes/cancelar.png</normaloff>Imagenes/cancelar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Factura">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>470</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
background-color: rgb(0, 51, 102);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Imprimir Factura</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Imagenes/imprimir.png</normaloff>Imagenes/imprimir.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue</string>
    </property>
    <property name="text">
     <string>Videojuegos Jericor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string>Catalogo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string>Tu cesta:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ProductosTabla">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>891</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genero</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Plataforma</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lanzamiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PEGI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Imagen</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="precioFinal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imagenes/mando.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
